### PR TITLE
Get instant users database query

### PR DESCRIPTION
**Pull request by laurieboyes**
Doesn't fire off any new events yet but worth merging early. Looks big and scary but mostly tests.

So when it gets a new content event, it calls `getUsers` with all that content's tags, which queries the myFT database for an array like this:

```
[
   { userId: 'some-user-id1', tags: ['some-concept-id', 'some-concept-id']},
   { userId: 'some-user-id2', tags: ['some-concept-id']}
   ...
]
```

From this we'll be able to create an event to send to `create-email`, choosing a tag to pass along (arbitrarily/intelligently tbc) and also passing along all the other content information from the event.

### CODE REVIEW

File: functions/get-instant-users/test/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+const expect = require('chai').expect;
+const sinon = require('sinon');
+const mockery = require('mockery');
+const uuid = require('node-uuid').v4;
+const invokeApex = require('../../../test/lib/invoke-apex');
+
+describe('Get instant users', () => {
```
Comment (laurieboyes) [inline functions/get-instant-users/test/index.spec.js]: This test file contains a lot of duplication, as I struggled to find a good way of swapping in different results for the `getUsers` mock for use in different test cases (e.g. failure handling). ⭐  Bonus fact: I hate mocking ⭐

File: functions/get-instant-users/test/lib/get-users.spec.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+			`MERGE (user1)-[:followed {instant: true}]->(concept0)`,
+			`MERGE (user1)-[:followed {instant: true}]->(concept1)`,
+
+			`MERGE (user2)-[:followed {instant: true}]->(concept1)`
+		].join('\n');
+
+		const query = `${conceptQueries} ${userQueries} ${followedInstantQueries}`;
+
+		console.log('query', query);
```
Comment (debugwand) [inline functions/get-instant-users/test/lib/get-users.spec.js]: stray console.log

File: functions/get-instant-users/test/lib/get-users.spec.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+require('dotenv').config({silent: true});
+
+const expect = require('chai').expect;
+const db = require('../../lib-shared/db').write;
```
Comment (debugwand) [inline functions/get-instant-users/test/lib/get-users.spec.js]: are you relying on a real db here?

Comment (laurieboyes) [inline functions/get-instant-users/test/lib/get-users.spec.js]: yep, relies on a local db running, which isn't ideal, but some day soon we might be moving to docker, which will make it much less not-ideal

**Comment by debugwand** [comment]
looks good

**Pull request merged**